policyengine_us/parameters/gov/states/ny/nysdps/clean_heat/cap.yaml

```diff
@@ -0,0 +1,12 @@
+description: New York State provides Clean Heat program incentives up to this percentage of qualified heat pump installation project cost.
+metadata:
+  unit: /1 
+  period: year
+  label: New York State Clean Heat qualified incentives rate cap
+  document: This cap applies to non-DAC sites for ConEd and to all sites for CH&E, NGRID, NYSEG, O&R, RG&E
+  reference:
+    - title: New York State Clean Heat Program Manual, CH&E, NGRID, NYSEG, O&R, RG&E (March 1, 2024)
+      href: https://cleanheat.ny.gov/assets/pdf/CHG&E%20NGrid%20NYSEG%20O&R%20and%20RG&E%20Program%20Manual_3.1.2024.pdf
```

Comment:
add pdf page to hyperlink where the value can be found 

E.g.
....`.pdf#page=x`

Reply:
all hyperlink references